collection-scripts/gather

```diff
@@ -78,38 +79,31 @@ case "$CLUSTER" in
     "HUB")
 
     check_managed_clusters
-    #oc adm inspect  ns/open-cluster-management  --dest-dir=must-gather
-    oc get pods -n "$HUBNAMESPACE" > ${BASE_COLLECTION_PATH}/gather-acm.log
-    oc get csv -n "$HUBNAMESPACE" > ${BASE_COLLECTION_PATH}/gather-acm.log 
-    oc adm inspect  ns/"$HUBNAMESPACE"  --dest-dir=must-gather
+    oc get pods -n "${OPERATORNAMESPACE}" > ${BASE_COLLECTION_PATH}/gather-acm.log
+    oc get csv -n "${OPERATORNAMESPACE}" > ${BASE_COLLECTION_PATH}/gather-acm.log
```

Comment:
What is the difference between this call and the `oc adm inspect csv` call below?

Reply:
I'm not totally sure. I had just left it in there based on its previous inclusion for the MCH. I would assume it's just getting a list of csv rather than doing a full inspection. Maybe it can be removed?